### PR TITLE
Add CameraTest model.

### PR DESCRIPTION
**Pull request by donmccurdy**
Tests viewports of ortho and perspective cameras. 

/cc #57 + https://github.com/mrdoob/three.js/pull/11706/

### CODE REVIEW

**Comment by lexaknyazev** [comment]
When  https://github.com/KhronosGroup/glTF-Blender-Exporter/issues/33 will be resolved, consider adding a camera with infinite projection.

File: 2.0/CameraTest/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Avocado
```
Comment (emackey) [inline 2.0/CameraTest/README.md]: copy-and-paste 😄

File: 2.0/CameraTest/glTF/CameraTest.gltf [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+                        0.0
+                    ], 
+                    "name" : "Ambient_Scene", 
+                    "type" : "ambient"
+                }
+            ]
+        }
+    }, 
+    "extensionsRequired" : [
+        "KHR_lights"
```
Comment (emackey) [inline 2.0/CameraTest/glTF/CameraTest.gltf]: Are lights required to test the cameras?

**Comment by donmccurdy** [comment]
Title is fixed and lights are removed, thanks!

Regarding unusually large dimensions, I think perhaps I should wait until we discuss conformance, before trying to define expected behavior on this test model.

**Pull request closed without merging**